chapter-customizing.asciidoc

```diff
@@ -1060,9 +1060,9 @@ output using +assertEquals()+.
 
 In +WeatherFormatterTest+, we used a utility from Apache Commons
 IO—the +IOUtils+ class. +IOUtils+ provides a number of helpful static
-functions that take most of the work out of input/output
+methods that take most of the work out of input/output
 operations. In this particular unit test, we used <methodname
-role="keep-together">IOUtils.toString()+ to copy the
+role="keep-together">+IOUtils.toString()+ to copy the
```

Comment:
This one still isn't rendering right: in the PDF output, that thing that looks like an HTML tag is being rendered as literal text that the reader can see.